protx-client/src/components/ProTx/components/dashboard/DisplaySelectors.jsx

```diff
@@ -78,12 +78,14 @@ function DisplaySelectors({
   geography,
   maltreatmentTypes,
   observedFeature,
+  analyticsType,
```

Comment:
add analyticsType and setAnalyticsType to proptypes. (probably as optional with default values of `null`)?